tools/isobuild/isopack-cache.js

```diff
@@ -118,6 +118,27 @@ _.extend(exports.IsopackCache.prototype, {
     });
   },
 
+  getSourceRoot(name, arch) {
+    const packageInfo = this._packageMap.getInfo(name);
+
+    if (packageInfo) {
+      if (packageInfo.kind === "local") {
```

Comment:
ah, so this is in a way the cause of the discrepancy between local and versioned packages in how they find files that are being imported.